vnet/lib/vnet/models/ip_lease.rb

```diff
@@ -60,6 +60,15 @@ def where_network_id(network_id)
 
     end
 
+    def valid_in_subnet
+      ipv4_prefix = Network[network_id][:ipv4_prefix]
+      IPAddress(convert_ipv4(Network[self.network_id][:ipv4_network], ipv4_prefix)).include? IPAddress(convert_ipv4(self.ipv4_address, ipv4_prefix))
+    end
+
+    def convert_ipv4(u32, prefix)
```

Comment:
__[reek]__
* [UtilityFunction] doesn't depend on instance state
* [UncommunicativeMethodName] has the name 'convert_ipv4'
* [UncommunicativeParameterName] has the parameter name 'u32'